tests/integration/common.go

```diff
@@ -625,3 +625,10 @@ func (s *CCVTestSuite) mustGetStakingValFromTmVal(tmVal tmtypes.Validator) (stak
 	s.Require().True(found)
 	return stakingVal
 }
+
+// nextBlocks moves `chain` forward by a `numberOfBlocks` blocks
+func nextBlocks(chain *ibctesting.TestChain, numberOfBlocks int64) {
```

Comment:
nit: all `nextBlocks` calls have the same `numberOfBlocks` arg value,
`i.e. nextBlocks(s.providerChain, s.providerApp.GetProviderKeeper().GetParams(s.providerCtx()).BlocksPerEpoch)`

Perhaps it's worth creating a wrapping method like
```func (s *CCVTestSuite) nextEpoch() {
nextBlocks(s.providerChain, s.providerApp.GetProviderKeeper().GetParams(s.providerCtx()).BlocksPerEpoch)`
}
```

Reply:
Good point. `nextEpoch` is much better. Fixed it.